static/css/global.css

```diff
@@ -59,5 +59,9 @@ html, body{
 }
 
 .min-height {
-    min-height: 30vw;
+    min-height: 30vh;
```

Comment:
I didnt notice this one before, my bad!
you are right here, the height should depend of the viewport height(vh), not the viewport width(vw)